libs/designer-ui/src/lib/token/inputToken.tsx

```diff
@@ -302,7 +306,9 @@ export const InputToken: React.FC<InputTokenProps> = ({ value, brandColor, icon,
     >
       <div className={styles.tokenIcon} style={iconStyle} aria-hidden="true" />
       {isSecure && <LockClosedRegular className={styles.secureIcon} aria-hidden="true" />}
-      <div className={styles.tokenTitle}>{title}</div>
+      <div className={styles.tokenTitle} style={{ maxWidth: '200px' }} title={title}>
+        {title}
+      </div>
```

Comment:
Where are we using this?

Reply:
oh this was just a sidequest when I noticed this in the External chat that tokens were too long in dictionary editor was changing the shape of the editor; but I was too deep in my parameters tab, but not yet complete to switch branches, so it was sorta snuck in